File: ClientApp/Config/Objects/Docs/Act.ts
import Laboratory from "../Items/Laboratory";
import Service from "../Items/Service";
import Client from "../Items/Client";

export default class Act
{
    public Laboratory: Laboratory;
    public Service: Service;
    //public Car: Car;
    public Client: Client;

    public Number: string;
    public SampleCloseDate: string;
    public CarGiveDate: string;
    public PPTOAddress: string;
    public NamePPTOExpert: string;
    
    constructor(Laboratory: Laboratory, Service: Service, Client: Client, Number: string,
                SampleCloseDate: string, CarGiveDate: string, PPTOAddress: string,
                NamePPTOExpert: string) 
    {
        this.Laboratory = Laboratory;
        this.Service = Service;
        this.Client = Client;
        this.Number = Number;
        this.SampleCloseDate = SampleCloseDate;
        this.CarGiveDate = CarGiveDate;
        this.PPTOAddress = PPTOAddress;
        this.NamePPTOExpert = NamePPTOExpert
    }
}

File: ServerApp/Config/Objects/Docs/Act.cs
using System.Collections.Generic;
using LBTDTools.ServerApp.Config.Objects.Docs.Properties.Act;
using LBTDTools.ServerApp.Config.Objects.Items;
using LBTDTools.ServerApp.Config.Objects.Items.Car;

namespace LBTDTools.ServerApp.Config.Objects.Docs
{
    public class Act
    {
        public Laboratory Laboratory { get; set; }
        public Service Service { get; set; }
        public Car Car { get; set; }
        public Client Client { get; set; }

        public VisualControl VisualControl = new();
        public WeightControl WeightControl = new();
        public COControl COControl = new();
        public SoundControl SoundControl = new();
        public DimensionsControl DimensionsControl = new();
        public SafeControl SafeControl = new();
        
        public string Number { get; set; }
        public string SampleCloseDate { get; set; }
        public string CarGiveDate { get; set; }
        public string PPTOAddress { get; set; }
        public string NamePPTOExpert { get; set; }
    }
}